src/main/scala/com/databricks/labs/overwatch/pipeline/SilverTransforms.scala

```diff
@@ -1314,4 +1314,119 @@ trait SilverTransforms extends SparkSessionWrapper {
 
   }
 
+  private def warehouseBase(auditRawDF: DataFrame): DataFrame = {
+    val warehouse_id_gen_w = Window.partitionBy('organization_id, 'warehouse_name)
+      .orderBy('timestamp).rowsBetween(Window.currentRow, 1000)
+    val warehouse_name_gen_w = Window.partitionBy('organization_id, 'warehouse_id)
+      .orderBy('timestamp).rowsBetween(Window.currentRow, 1000)
+    val warehouse_id_gen = first('warehouse_id, true).over(warehouse_id_gen_w)
+    val warehouse_name_gen = first('warehouse_name, true).over(warehouse_name_gen_w)
+
+    val warehouseSummaryCols = auditBaseCols ++ Array[Column](
```

Comment:
please make the timestamp field unixTimeMS as it is in other silver tables and make timestamp == an actual timestamp type for humans. The key for this and the gold target should be switched to the unixTimeMS field.

Reply:
As discussed , we will keep timestamp field in Silver table. As other Silver tables has timestamp  column too.